src/gen.c

```diff
@@ -1631,6 +1631,7 @@ static int do_help(int argc, char **argv)
 		"\n"
 		"       " HELP_SPEC_OPTIONS " |\n"
 		"                    {-L|--use-loader} }\n"
+		"                    {-B|--base-btf} }\n"
```

Comment:
I'm just seeing you need to update the line above, here, to remove the ending brace. In other words, instead of this:

```
		"                    {-L|--use-loader} }\n"
		"                    {-B|--base-btf} }\n"
```

Having this:

```
		"                    {-L|--use-loader}\n"
		"                    {-B|--base-btf} }\n"
```

Reply:
good catch